staging/src/k8s.io/client-go/discovery/cached/disk/cached_discovery_test.go

```diff
@@ -786,6 +786,10 @@ func (d *fakeDiscoveryClient) OpenAPIV3() openapi.Client {
 	panic("unimplemented")
 }
 
+func (d *fakeDiscoveryClient) WithLegacy() discovery.DiscoveryInterface {
+	panic("unimplemented")
```

Comment:
shouldn't this return d if it doesn't support legacy-only mode per the godoc? panicking here makes the fake client unusable for callers that use WithLegacy(), right?

Reply:
Agreed 👍

Reply:
actually, since this type is local to this test, I don't think this instance matters

the staging/src/k8s.io/client-go/discovery/fake/discovery.go one matters more since that is exported for use in other packages' tests